### PR TITLE
Changed contact form to go to info email

### PR DESCRIPTION
**Pull request by pjsier**

### CODE REVIEW

**Comment by karbassi** [comment]
@pjsier I wouldn't hardcode this in there. I would suggest creating a variable in the settings.py file.

**Comment by pjsier** [comment]
Just changed that in the updated commit, does that work? Good call

**Comment by karbassi** [comment]
Looks good to me :+1:

File: coderdojochi/views.py [hunk 1]
```diff
@@ -1021,7 +1021,7 @@ def contact(request, template_name="contact.html"):
                     subject='CoderDojoChi | Contact Form Submission',
                     body='Contact Form Submission from ' + request.POST['name'] + ' (' + request.POST['email'] + '). ' + request.POST['body'],
                     from_email=request.POST['email'],
-                    to=[v for k,v in settings.ADMINS]
+                    to=settings.CONTACT_EMAIL
```
Comment (bkellgren) [inline coderdojochi/views.py]: `to` should be a list here  i.e. `to=[settings.CONTACT_EMAIL]`

**Comment by bkellgren** [comment]
one comment that should be updated, otherwise looks good :+1:

**Pull request merged**